pipes/rules/common.rules

```diff
@@ -15,9 +15,15 @@ def set_env_vars():
 
 def read_tab_file(fname):
     with open(fname, 'rt') as inf:
-        header = [item.strip() for item in inf.readline().rstrip('\n').split('\t')]
+        header = [item.strip() for item in inf.readline().strip().rstrip('\n').split('\t')]
```

Comment:
let's see, while this new parsing of the header will lose any trailing empty columns (e.g. \t\t\t\n) in theory, the header is the one place where we shouldn't allow that anyway, so that's okay... trailing empty columns look good in the data rows below. It also handles trailing spaces within the interior columns itself. The csv case above is okay because the delimiter is not affected by strip()...